server/superdesk/publish/odbc_test.py

```diff
@@ -15,7 +17,7 @@
 
 
 class ODBCTests(TestCase):
-    subscribers = [{"_id": "1", "name": "Test", "can_send_takes_packages": False, "media_type": "media",
+    subscribers = [{"_id": "1", "name": "Test", "subscriber_type": SUBSCRIBER_TYPES.WIRE, "media_type": "media",
```

Comment:
We need something to determine media client like Fairfax, Guardian and
non-media like Commbank, Myer for embargo stories or something similar. This
was few weeks ago.

On Wednesday, 8 July 2015, Siva Kuna <notifications@github.com> wrote:

> In server/superdesk/publish/odbc_test.py
> <https://github.com/superdesk/superdesk/pull/808#discussion_r34113360>:
>
> > @@ -15,7 +17,7 @@
> >
> >
> >  class ODBCTests(TestCase):
> > -    subscribers = [{"_id": "1", "name": "Test", "can_send_takes_packages": False, "media_type": "media",
> > +    subscribers = [{"_id": "1", "name": "Test", "subscriber_type": SUBSCRIBER_TYPES.WIRE, "media_type": "media",
>
> @akintolga <https://github.com/akintolga> I checked with Andrew on this.
> He is not sure if Media Type becomes redundant after introducing Subscriber
> Type. He wants you to confirm on this part. If you say Media Type is
> redundant then I will remove the property from both Frondend and Backend.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/superdesk/superdesk/pull/808/files#r34113360>.
>


-- 

*Mayur Dhamanwala | *Senior Analyst Programmer


*Australian Associated Press*
Level 5, 3 Rider Boulevard, Rhodes NSW 2138, Australia
PO Box 3411, Rhodes NSW 2138, Australia
*T *+61 2 9322 8582  *F *+61 2 9322 8086
mdhamanwala@aap.com.au | www.aap.com.au

-- 
*This email may contain information that is confidential. If you receive an 
email in error please delete it immediately.*